google-cloud-core/lib/google/cloud.rb

```diff
@@ -156,9 +156,12 @@ def self.loaded_files
     # @private
     #
     def self.auto_load_files
-      return Gem.find_latest_files "google-cloud-*.rb" if Gem.respond_to? :find_latest_files
-      # Ruby 2.0 does not have Gem.find_latest_files
-      Gem.find_files "google-cloud-*.rb"
+      Gem.loaded_specs.values.filter_map do |spec|
+        next unless spec.name.start_with?("google-cloud-")
```

Comment:
The CI failures are rubocop warnings. To conform with this library's style, omit the method-argument parentheses on this line and the following two lines.